src/main/java/org/carlmontrobotics/robotcode2023/subsystems/Arm.java

```diff
@@ -1,6 +1,48 @@
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.ARM_MASS_KG;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.ARM_TELEOP_MAX_GOAL_DIFF_FROM_CURRENT_RAD;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.ARM_UPPER_LIMIT_RAD;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.BACK;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.COM_ARM_LENGTH_METERS;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.COM_ROLLER_LENGTH_METERS;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.CONE;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.CUBE;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.DT_EXTENSION_FOR_ROLLER;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.DT_TOTAL_WIDTH;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.MAX_FF_ACCEL;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.MAX_FF_VEL;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.ROLLER_COM_CORRECTION_RAD;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.ROLLER_LENGTH_METERS;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.ROLLER_MASS_KG;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.SAFE_HEIGHT;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.WRIST;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.WRIST_CURRENT_LIMIT_AMP;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.WRIST_DISCONTINUITY_RAD;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.WRIST_LOWER_LIMIT_RAD;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.WRIST_UPPER_LIMIT_RAD;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.armConstraints;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.armMotorPort;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.g;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.inverted;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.kA;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.kD;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.kG;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.kI;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.kP;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.kS;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.kV;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.offsetRad;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.posToleranceRad;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.rotationToRad;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.velToleranceRadPSec;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.wristConstraints;
+import static org.carlmontrobotics.robotcode2023.Constants.Arm.wristMotorPort;
```

Comment:
Just use a `*` import

Reply:
Imo, it helps to limit the number of imports required to use wildcard on Organize Imports in vscode settings. Personally, I set `java.sources.organizeImports.staticStarThreshold` to `2`.